README.md

```diff
@@ -1,116 +1,67 @@
-          }
-        ],
-        "packages": [ "@Core", "grub2-pc", "httpd" ]
-        }
-    },
-    {
-      "name": "org.osbuild.systemd",
-      "options": {
-        "enabled_services": [ "httpd" ]
-      }
-    },
-    {
-      "name": "org.osbuild.grub2",
-      "options": {
-        "root_fs_uuid": "76a22bf4-f153-4541-b6c7-0332c0dfaeac"
-      }
-    }
-  ],
-  "assembler": {
-    "name": "org.osbuild.qemu",
-    "options": {
-      "format": "qcow2",
-      "filename": "example.qcow2",
-      "ptuuid": "0x7e83a7ba",
-      "root_fs_uuid": "76a22bf4-f153-4541-b6c7-0332c0dfaeac",
-      "size": 3221225472
-    }
-  }
-}
-```
+OSBuild
+=======
 
-`osbuild` runs each of the stages in turn, isolating them from the host and
-from each other, with the exception that they all operate on the same
-filesystem-tree. The assembler is similarly isolated, and given the same
-tree, in read-only mode and assembles it into an image without altering
-its contents.
+Build-Pipelines for Operating System Artifacts
 
-The filesystem tree produced by the final stage of a pipeline, is named
-and optionally saved to be reused as the base for future pipelines.
+OSBuild is a pipelined build system for operating system artifacts. It defines
+a universal pipeline description and a build system to execute them, producing
+artifacts like operating system images, working towards an image build pipeline
+that is more comprehensible, reproducible, and extendable.
 
-Each stage is passed the (appended) `options` object as JSON over stdin.
+See the `osbuild(1)` man-page for details on how to run osbuild, the definition
+of the pipeline description, and more.
 
-The above pipeline has no base and produces a qcow2 image.
+### Project
 
-## Running
+ * **Website**: <https://www.osbuild.org>
+ * **Bug Tracker**: <https://github.com/osbuild/osbuild/issues>
 
-```
-usage: __main__.py [-h] [--build-env ENV] [--store DIRECTORY] [-l DIRECTORY]
-                   [--json]
-                   PIPELINE
-
-Build operating system images
-
-positional arguments:
-  PIPELINE              json file containing the pipeline that should be
-                        built, or a '-' to read from stdin
-
-optional arguments:
-  -h, --help            show this help message and exit
-  --build-env ENV       json file containing a description of the build
-                        environment
-  --store DIRECTORY     the directory where intermediary os trees are stored
-  --sources SOURCES     json file containing a dictionary of source
-                        configuration
-  -l DIRECTORY, --libdir DIRECTORY
-                        the directory containing stages, assemblers, and the
-                        osbuild library
-  --json                output results in JSON format
-```
+### Requirements
+
+The requirements for this project are:
+
+ * `python >= 3.8`
```

Comment:
Isn't it Python 3.7?

Reply:
It is indeed. Thanks, totally missed that.

Reply:
Fixed up locally.

Btw. these versions are "best efforts" and document what works for me. I did not verify all of them. Completely forgot to mention that. But I thought mentioning a version that works would be better than none. But no strong opinions on that.

Reply:
Best effort makes sense. Should document the earliest that is known to work (but doesn't mean that earlier might not). In the case of python we also should keep in mind not to start depending on 3.8 by accident, as that might cause some pain.

Reply:
Indeed, but 3.7 definitely works and what we're using in CI. 3.8 is quite new and we probably don't want to depend on it for a while.